src/js/view/controls/components/chapters.mixin.js

```diff
@@ -53,7 +52,7 @@ const ChaptersMixin = {
             return;
         }
 
-        _.each(this.cues, (cue) => {
+        this.cues.forEach((cue) => {
```

Comment:
Will `this.cues` always be defined? I know `_.each` had guard checks against this.

Reply:
It is set `[]` in TimeSlider constructor.